src/ray/object_manager/plasma/protocol.h

```diff
@@ -41,6 +41,13 @@ using flatbuf::MessageType;
 using flatbuf::ObjectSource;
 using flatbuf::PlasmaError;
 
+inline constexpr std::string_view DEBUG_STRING = "debug_string";
```

Comment:
```suggestion
inline constexpr std::string_view kDebugString = "debug_string";
```

Actually don't sure whether we need to define a constant here, seems it's just used in one place.

Reply:
It should also be defined in the protocol.cc file

Reply:
> Actually don't sure whether we need to define a constant here, seems it's just used in one place.

Although it is only used in one place but the function can potentially be called multiple times, so I think it is still better to put it here like this.